src/UIAutomation/Microsoft.UI.UIAutomation/RemoteOperationInstructions.h

```diff
@@ -933,6 +939,24 @@ struct GetMetadataValue
     OperandId metadataId;
 };
 
+struct CallExtension
+{
+    constexpr static InstructionType type = InstructionType::CallExtension;
+
+    OperandId targetId;
+    OperandId extensionIdId;

```

Comment:
>OperandId extensionIdId; [](start = 4, length = 24)

Should this be just `extensionId`?

Reply:
I think this is OK. This follows the same pattern as `propertyIdId`, i.e. in this case, that field represents the operand ID of the extension ID parameter.

Reply:
yes, that makes sense.